src/components/AwardCard.vue

```diff
@@ -20,8 +20,14 @@
 export default {
   name: 'AwardCard',
   props: {
-    username: String,
-    place: Number,
+    username: {
+      type: String,
+      default: '',
+    },
+    place: {
+      type: String,
+      default: '',
```

Comment:
I think the change to object syntax is great. This was previously a Number, was it changed deliberately? I also think that we should make `place` a required prop instead of giving it a default value, because it doesn't really make sense to use this component without specifying a particular place
```suggestion
      type: Number,
      required: true,
```

Reply:
oups yes sorry, wasn't done intentionally